pom.xml

```diff
@@ -860,6 +860,7 @@
                             <goal>verify</goal>
                         </goals>
                         <configuration>
+                            <reuseForks>false</reuseForks>
```

Comment:
Yeah, otherwise the second Integration test class will throw an error, it's an workaround and it will only affect integration tests since it's in the different plugin. https://github.com/arquillian/arquillian-container-glassfish/issues/24